src/components/dialog-test.js

```diff
@@ -112,6 +126,19 @@ describes.realWin('Dialog', {}, env => {
       expect(graypaneStubs.show).to.be.calledOnce.calledWith(ANIMATE);
     });
 
+    it('should build the view and show hidden iframe', function* () {
+      const openedDialog = yield dialog.open(NO_ANIMATE, HIDDEN);
+      yield openedDialog.openView(view);
+      expect(computedStyle(win, element)['opacity']).to.equal('1');
```

Comment:
Let's test that dialog's element is also now visible?

Reply:
start

Reply:
Done